_data/taps/versions/bing-ads.yml

```diff
@@ -0,0 +1,161 @@
+              <li>FinalUrl</li>
+              <li>FinalMobileUrl</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td class="attribute-name">adgroup_performance_report</td>
+          <td class="attribute-description">
+            <ul>
+              <li>HistoricQualityScore</li>
+              <li>HistoricExpectedCtr</li>
+              <li>HistoricAdRelevance</li>
+              <li>HistoricLandingPageExperience</li>
+              <li>Status</li>
+            </ul>
+          </td>
+          <td class="attribute-description">
+            <ul>
+              <li>HistoricalQualityScore</li>
+              <li>HistoricalExpectedCtr</li>
+              <li>HistoricalAdRelevance</li>
+              <li>HistoricalLandingPageExperience</li>
+              <li>CampaignStatus</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td class="attribute-name">campaign_performance_report</td>
+          <td class="attribute-description">
+            <ul>
+              <li>HistoricQualityScore</li>
+              <li>HistoricExpectedCtr</li>
+              <li>HistoricAdRelevance</li>
+              <li>HistoricLandingPageExperience</li>
+              <li>Status</li>
+            </ul>
+          </td>
+          <td class="attribute-description">
+            <ul>
+              <li>HistoricalQualityScore</li>
+              <li>HistoricalExpectedCtr</li>
+              <li>HistoricalAdRelevance</li>
+              <li>HistoricalLandingPageExperience</li>
+              <li>CampaignStatus</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td class="attribute-name">keyword_performance_report</td>
+          <td class="attribute-description">
+            <ul>
+              <li>FinalAppURL</li>
+              <li>FinalURL</li>
+              <li>FinalMobileURL</li>
+              <li>HistoricQualityScore</li>
+              <li>HistoricExpectedCtr</li>
+              <li>HistoricAdRelevance</li>
+              <li>HistoricLandingPageExperience</li>
+              <li>SidebarBid</li>
+            </ul>
+          </td>
+          <td class="attribute-description">
+            <ul>
+              <li>FinalAppUrl</li>
+              <li>FinalUrl</li>
+              <li>FinalMobileUrl</li>
+              <li>HistoricalQualityScore</li>
+              <li>HistoricalExpectedCtr</li>
+              <li>HistoricalAdRelevance</li>
+              <li>HistoricalLandingPageExperience</li>
+              <li>FirstPageBid</li>
+            </ul>
+          </td>
+      </table>
+
+  - date: "October 4, 2018"
+    version: "1.0"
+    type: "version-deprecation"
+    content: |
+      This version will be sunset on or before October 31, 2018, in accordance with Microsoft's announcement to [deprecate version 11 of the Bing Ads API](https://docs.microsoft.com/en-us/bingads/guides/migration-guide?view=bingads-12){:target="new"}.
+
+  - date: "July 31, 2018"
```

Comment:
Is this supposed to be here? Looks like it was copied from a DB tap with `logical replication` mentioned.

Reply:
Corrected